pages/hold/request/[id]/edd.tsx

```diff
@@ -158,7 +155,7 @@ export default function EDDRequestPage({
         <HoldItemDetails item={item} />
         {isLoading || formPosting ? (
           <SkeletonLoader showImage={false} data-testid="edd-request-loading" />
-        ) : isEddAvailable ? (
+        ) : status !== "unavailable" ? (
```

Comment:
There also might be some test coverage missing if this is not triggering a test failure.

Reply:
I'm surprised the linter doesn't pick this up.

Reply:
I think it's normal for the linter not to get this since it's syntactically correct, but could be undefined. I agree that this needs better test coverage though.